src/main/java/org/opencds/cqf/tooling/cql_generation/drool/adapter/DefinitionAdapter.java

```diff
@@ -0,0 +1,234 @@
+package org.opencds.cqf.tooling.cql_generation.drool.adapter;
```

Comment:
Generally, an "adapter" is used to connect one interface to another. In this case a whole new object is being created, so I'd suggest using "builder" or "converter" instead. Same applies throughout.